test/models/track_test.rb

```diff
@@ -16,6 +16,14 @@ class TrackTest < ActiveSupport::TestCase
     assert_equal track, Track.for!(track.slug)
   end
 
+  test "#active scope" do
+    # Create one active and one inactive track
```

Comment:
I think we can remove this comment.

Reply:
The reason this comment is there is that L22 creates a track but doesn't do anything with it. Someone could accidentally delete that line, the test would still pass, but the actual usefulness of the test would then be lost. So by adding the comment I wanted to emphasise that two tracks should be created, and the second line isn't some sort of mistake etc.

Is there a better way to achieve this without a comment?

Reply:
Would assigning the result to an unused variable be better?

Reply:
Assert that the other track is not included in the scope.